Application/src/view/visuals/Draw.java

```diff
@@ -219,6 +204,17 @@ void init(GraphicsContext gc) {
         gc.drawImage(animatedBackground,0,0, this.canvas.getWidth(), this.canvas.getHeight());
     }
 
+    /**
+     * Initializes the drawing of class diagram
+     * @param gc
+     */
+
+    void initClassDiagram(GraphicsContext gc){
```

Comment:
I gave it a quick try, and seems that the backhground of the class diagram overwrites the sequence background. If its not to bad, could we keep it this way? Otherwise I can maybe dig deaper and try to generalize it